### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Create Web Site_Screen Reader: Narrator/NVDA is reading toggle buttons as "checkbox".

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#132]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA

**MAS ID Violated**: MAS1.3.1

**Repro Steps:** 
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "Create Web Site" expandable button and Press Enter.
6. Tab till "Select Folder" button and Press Enter.
7. Navigate to "Application Pool" page tab.
8. Verify that screen reader is reading the proper description of the toggle button.

**Actual Result:**
Narrator/NVDA is reading toggle buttons as "checkbox"

**Expected Result:**
Screen Reader should read it as: Toggle button, selected Yes/No.

**User Impact:**
Since checkbox and toggle buttons are two different types of controls and they work differently, so screen reader should read these properly. User may expect multiple options if screen reader announce it as checkbox, whereas toggle button can only provide the boolean values.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment:**
[Bug 132_screen reader is reading the toggle buttons as checkbox.pptx](https://github.com/Microsoft/IIS.WebManager/files/2327807/Bug.132_screen.reader.is.reading.the.toggle.buttons.as.checkbox.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
